web_src/js/features/repo-search.js

```diff
@@ -0,0 +1,15 @@
+export function initRepositorySearch() {
+  const repositorySearchForm = document.querySelector('#repo-search-form');
+  if (!repositorySearchForm) return;
+
+  for (const radio of repositorySearchForm.querySelectorAll('input[type=radio]')) {
+    radio.addEventListener('click', (ev) => {
+      ev.preventDefault();
```

Comment:
```suggestion
    radio.addEventListener('click', (e) => {
      e.preventDefault();
```

Reply:
Done with https://github.com/go-gitea/gitea/pull/29231/commits/53108435b487b6f243068ee95328f411da5f5316
But why use single-letter variable names?

Reply:
No real reason besides that it's a habit and commonplace in this repo.